api/database/migrations/2023_04_24_195500_experience_evidence_setup.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('pool_candidate_education_requirement_experience', function (Blueprint $table) {
+            $table->uuid('id')->primary('id')->default(new Expression('gen_random_uuid()'));
+            $table->uuid('pool_candidate_id')->nullable(false);
+            $table->foreign('pool_candidate_id', 'pool_candidate_foreign')->references('id')->on('pool_candidates');
```

Comment:
Yeah, this explicit name seems to hav solved the issue.  Maybe the autogenerated name was just too long?  :laughing:
Let's try to make it a little more descriptive though.  Maybe `pool_candidate_education_requirement_experience_foreign`?

Reply:
I dunno, that looks a bit odd to me sticking "foreign" at the end of the table name, especially since that'd be different from the rest of our pivot tables

Reply:
Yeah, autogenerated foreign id names can get overly long. @vd1992, I think @petertgiles is suggesting a new name for the foreign id constraint, not the table, ie
```suggestion
            $table->foreign('pool_candidate_id', 'pool_candidate_education_requirement_experience_foreign')->references('id')->on('pool_candidates');
```

Reply:
Oh yeah, that makes more sense 😊